packages/debugger/test/data/returnvalue.js

```diff
@@ -75,7 +75,7 @@ describe("Return value decoding", function () {
   var compilations;
 
   before("Create Provider", async function () {
-    provider = Ganache.provider({seed: "debugger", gasLimit: 7000000});
+    provider = Ganache.provider({legacyInstamine: true, vmErrorsOnRPCResponse: true, seed: "debugger", gasLimit: 7000000});
```

Comment:
These tests rely on `vmErrorsOnRPCResponse: true`, which is no longer the default in Ganache v3